### ISSUE TITLE
Textual Inversion giving ..grinning lego clowns from hell

### ISSUE THREAD

**Issue by SorenTruelsen** [TheLastBen/fast-stable-diffusion#1032]
Hi Ben and the team
Been trying to use textual inversion for a couple of days (around 100 colab-units). All my embeddings give "Yellow Lego Clown Figures". 

I have done loads of different variations to try to find a mistake in my setup; used different learning rates, initialization text, number of vectors, picture sets, prompts, training steps. At around 25K they solidify in this ghastly horror expression. 

The setting in GUI for the attached image was:
Initialization text: Woman posing secretary
Vectors: 3
Learning rate: 0.005
Batch size: 1
Gradient accumulation steps: 1
Choose latent sampling method: Once 

Not sure if its a bug, but I really haven't been able to locate anything to explain this artifact. A suggestion from Discord was it was related to CFG setting or Sampler, but since the GUI is not really offering to change this (other than for the preview image), I am not sure how to play around with that. 

Thanks

![image](https://user-images.githubusercontent.com/7306254/207812772-d3e21bb0-129b-4ca5-87e6-713488650b90.png)

**Comment by SorenTruelsen** [comment]
Moved to Discussions
